Makefile

```diff
@@ -59,7 +59,7 @@ test: ## run tests quickly with the default Python
 
 test-nb: ## run tests quickly with the default Python
 	#py.test for notebooks
-	pytest --nbval $(CURDIR)/notebooks/demo --sanitize-with $(CURDIR)/notebooks/output_sanitize.cfg --ignore $(CURDIR)/notebooks/*/.ipynb_checkpoints
```

Comment:
The reason for keeping this is to ensure that if the Jupyter kernel creates checkpoints. those won't be tested. Just want to be sure that this change is desired.

Reply:
The behavior of `--ignore` was not as expected. When I try it locally it scans *all* notebooks (demo, examples, ...) except those in the ignore list. Without the ignore at least it worked.

Reply:
... maybe `--ignore` conflicts with the `--nbval` strategy. Looks like if I use `--nbval` only it ignores `.ipynb_checkpoints` by default.

Reply:
Awesome. I imagine that that's a new feature. Looks good to me.